std/experimental/allocator/building_blocks/region.d

```diff
@@ -356,9 +363,12 @@ struct Region(ParentAllocator = NullAllocator,
     // Create a 64 KB region allocated with malloc
     auto reg = Region!(Mallocator, Mallocator.alignment,
         Yes.growDownwards)(1024 * 64);
+
+    assert((() pure nothrow @safe @nogc => reg.empty)() ==  Ternary.yes);
```

Comment:
Since the unit test is explicitly marked as `@system`, it would probably have been better to put these in their own unit test block and mark it `pure nothrow @safe @nogc`.